config.yaml

```diff
@@ -25,4 +25,14 @@ options:
 
       Example usage:
       8080: "expose"
-      8443: ""
\ No newline at end of file
+      8443: ""
+  docker-registry:
+    type: string
+    default: ""
+    description: |
+      Private Docker registry FQDN. Needed when deploying to Kubernetes and image is behind a private repository.
```

Comment:
Currently these options are only used by Kubernetes, maybe they will find a use in the layer-image-host.